.dockerignore

```diff
@@ -0,0 +1 @@
+.venv

```

Comment:
ignore result.png and other files as well

Reply:
Fixed